contracts/ubeswap/contracts/uniswapv2/interfaces/IUniswapV2Router02SwapOnly.sol

```diff
@@ -3,7 +3,7 @@
 pragma solidity ^0.8.3;
 
 /// @notice Swaps tokens
-interface IUbeswapRouter {
+interface IUniswapV2Router02SwapOnly {
```

Comment:
This name was changed to reflect the fact that contracts that implement this interface are really just restrictions of the normal `IUniswapV2Router02` interface that only support swaps. Removing the "`Ubeswap`" naming scheme means we can use this in place of `IUniswapV2Router02` when we only care about swapping.